modules/gapi/src/api/freetype_mask_creator.hpp

```diff
@@ -31,10 +31,18 @@ class FreeTypeBitmaskCreator : public IBitmaskCreator
 public:
     FreeTypeBitmaskCreator(const std::string& path) : m_font_path(path)
     {
+        if (FT_Init_FreeType(&m_library) ||
+            FT_New_Face(m_library, m_font_path.c_str(), 0, &m_face))
+        {
+            m_is_initiliazed = false;
+        }
```

Comment:
what you basically need is just
```c++
if (FT_Init_FreeType(&m_library) != 0)
{
   throw "Failed to initialize FT";
}
if (FT_New_Face(...) != 0)
{
   FT_Done_FreeType(...);
   throw "Failed to create a font face";
}
```

Please note I omitted an exception type for brevity but hope you'll address that.

Reply:
Fixed

Reply:
...waiting for an update :)